docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+# Oled Display
+Extension to use for your build with OLED Display
+
+If you need help soldering, there's a small note on the end to help.
+
+# Preparation
```

Comment:
```suggestion
## Preparation
```
Don't skip heading hierarchies (*H1* / *#* is always followed by *H2* / *##*, and so on), in this case use a second order heading.